app/src/main/java/org/simple/clinic/activity/TheActivity.kt

```diff
@@ -68,7 +73,8 @@ class TheActivity : AppCompatActivity() {
 
   override fun attachBaseContext(baseContext: Context) {
     setupDiGraph()
-    val contextWithRouter = wrapContextWithRouter(baseContext)
+    val contextWithOverridenLocale = LocaleOverrideContextWrapper.wrap(baseContext, locale)
```

Comment:
Nitpick 😛 It is `overridden` with double d.

Reply:
Whoops! will fix!